js/models/purchase/Order.js

```diff
@@ -23,30 +23,31 @@ export default class extends BaseModel {
     };
   }
 
+  addAddress(sAddr) {
```

Comment:
The address from settings is not in the same format as the address needed here, so this method will translate it and add it to the order.